lib/openscap_parser/rule.rb

```diff
@@ -20,11 +20,13 @@ def title
     end
 
     def description
-      @description ||= @rule_xml.at_css('description').text.delete("\n")
+      rule_node ||= @rule_xml.at_css('description')
+      @description ||= newline_to_whitespace(rule_node.text) if rule_node && rule_node.text
     end
 
     def rationale
-      @rationale ||= @rule_xml.at_css('rationale').children.text.delete("\n")
+      rationale_node ||= @rule_xml.at_css('rationale')
+      @rationale ||= newline_to_whitespace(rationale_node.children.text) if rationale_node && rationale_node.children && rationale_node.children.text
```

Comment:
This line is too long. I guess we still need to turn on rubocop in this repo :see_no_evil: